### DISCUSSION TITLE
gpg: The key that signed this doesn't have usage flags that allow signing

### DISCUSSION THREAD

**Discussion by ncw** [community/community#33072] [Pull Requests]
I can't get my commits to be verified.

I have uploaded my GPG key, and git thinks the commits are signed locally

```
$ git verify-commit 1ef15136d8d5baa670e9c3137c26406f57cdc514
gpg: Signature made Wed 14 Sep 2022 18:21:21 BST
gpg:                using DSA key FBF737ECE9F8AB18604BD2AC93935E02FF3B54FA
gpg:                issuer "nick@craig-wood.com"
gpg: Good signature from "Nick Craig-Wood <nick@craig-wood.com>" [ultimate]
```

However in my pull request they appear to be unverified and have this message on a popup

> The key that signed this doesn't have usage flags that allow signing

https://github.com/jlaffaye/ftp/pull/281/commits/1ef15136d8d5baa670e9c3137c26406f57cdc514

![image](https://user-images.githubusercontent.com/536803/190224051-a3026a40-e476-476a-92cc-24a6f7b762c2.png)

Any ideas?

**Comment by airtower-luna** [comment]
That error message is odd, but from a glance at the key (1024 bit DSA) I assume it's simply categorized as too weak to trust for a signature. :sweat_smile:
